desktop/src/com/mygdx/game/client/screen/GameRoomScreen.java

```diff
@@ -0,0 +1,77 @@
+package com.mygdx.game.client.screen;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.ScreenAdapter;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.mygdx.game.client.di.StageModule;
+import com.mygdx.game.client.ui.PlayerRoomDialogFactory;
+import com.mygdx.game.client_core.model.NetworkJobsQueueJobJobberManager;
+import com.mygdx.game.client_core.network.QueueMessageListener;
+import com.mygdx.game.client_core.network.service.GameStartService;
+import com.mygdx.game.core.network.messages.GameStartedMessage;
+import dagger.Lazy;
+import lombok.NonNull;
+import lombok.extern.java.Log;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+@Singleton
+@Log
+public class GameRoomScreen extends ScreenAdapter {
+
+  private final Stage stage;
+  private final Lazy<GameScreen> gameScreenBuilder;
+  private final GameStartService gameStartService;
+  private final PlayerRoomDialogFactory roomDialogFactory;
+  private final QueueMessageListener connection;
+  private final NetworkJobsQueueJobJobberManager jobManager;
+
+  @Inject
```

Comment:
tu jest sam pokój:
jak na razie to jest tylko to co zawsze, czyli "start game", ale do dodania to może jakiś guzik "Add bot player", który na serwerze uruchamiałby osobny proces z botem, albo jakoś i tak i on by się łączył wtedy jako gracz

można dodać właśnie tę listę graczy i jakieś checki czy wszyscy gracze są gotowi i opcje do mapy - na razie na sztywno jest mapa o wielkości 5x5 z typem 401, który jest z configów

może da się pobrać te configi i wyświetlić z nich listę z której będzie można wybrać typ mapy